target/linux/ath79/image/generic.mk

```diff
@@ -1994,6 +1994,18 @@ define Device/kuwfi_n650
 endef
 TARGET_DEVICES += kuwfi_n650
 
+define Device/kuwfi_wb5g08
+  SOC := qca9563
+  DEVICE_VENDOR := KuWfi
+  DEVICE_MODEL := WB 5G08
+  IMAGE_SIZE := 16000k
+  IMAGES += tftp.bin
+  IMAGE/tftp.bin := $$(IMAGE/sysupgrade.bin) | yuncore-tftp-header-16m
+  DEVICE_PACKAGES := kmod-switch-ar8xx kmod-ath10k swconfig \
```

Comment:
`swconfig` is a default package, we don't need to add it again. And `kmod-switch-ar8xx` is an useless empty package on ath79 target.

Reply:
fixed